README.md

```diff
@@ -21,41 +21,63 @@ repositories {
 dependencies {
 // [...]
     // Coppernic
-    implementation 'fr.coppernic.sdk.cpcutils:CpcUtilsLib:6.14.0'
-    implementation 'fr.coppernic.sdk.core:CpcCore:1.3.0'
-    implementation 'fr.coppernic.sdk.pcsc:CpcPcsc:1.5.0'
+    implementation 'fr.coppernic.sdk.cpcutils:CpcUtilsLib:6.19.1'
+    implementation 'fr.coppernic.sdk.core:CpcCore:2.1.12'
+    implementation 'fr.coppernic.sdk.pcsc:CpcPcsc:1.5.4'
 // [...]
 }
 
 ```
 
 ### Power management
 
- * Implements power listener
+ * Define used peripheral
+
+ Each terminal (COne, Access-ER, IdPlatform), can contain one or more Pcsc reader.
+ Check their availability in technical specifications.
+
+https://www.coppernic.fr/c-one-2-e-id/
+https://www.coppernic.fr/access-er/
+https://www.coppernic.fr/access-er-e-id/
+https://www.coppernic.fr/id-platform/
+
+```java
+
+    private Peripheral getPeripheral() {
+        if (OsHelper.isCone()) {
+            return ConePeripheral.RFID_ELYCTIS_LF214_USB;
```

Comment:
We need to provide a solution to power on  C-One with CR30 and Microchip

Reply:
Tu veux dire qui soit indépendante du lecteur qui est derrière ?